sample_consensus/include/pcl/sample_consensus/impl/ransac.hpp

```diff
@@ -73,8 +73,6 @@ pcl::RandomSampleConsensus<PointT>::computeModel (int)
   const double log_probability  = std::log (1.0 - probability_);
   const double one_over_indices = 1.0 / static_cast<double> (sac_model_->getIndices ()->size ());
 
-  unsigned skipped_count = 0;
```

Comment:
The changes here in ransac.hpp are not correct: All threads have to share the same `skipped_count`. Making it thread-local changes the behaviour. I would suggest to revert all changes in ransac.hpp.

Reply:
Missing a Unit test on the OMP version then?

Reply:
Would be nice to test that, but there is no way to get the value of `skipped_count` after `computeModel` ran, and I don't think it is worth to change the API of RANSAC just so we can write a test for this.

Reply:
So the behavioral change doesn't produce different results?

Reply:
As I see it, RANSAC would take more time in certain situations. Let's say `max_skip` is 10000. In the current state of the code (master), `computeModelCoefficients` would fail 10000 times overall before RANSAC terminates (no matter how many threads there are). With these changes, `computeModelCoefficients` would fail 10000 time _per thread_  before each thread (and eventually RANSAC) terminates.